app/assets/javascripts/components/notes/Note.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class Note extends React.Component {
+  constructor(props) {
+    super(props);
+    this._handleDelete = this._handleDelete.bind(this);
+  }
+
+  render() {
```

Comment:
To follow the standard of the community, let's keep this order of methods inside a component https://github.com/airbnb/javascript/tree/master/react#ordering